tests/libYARP_math/MathTest.cpp

```diff
@@ -662,7 +664,13 @@ class MathTest : public UnitTest {
         // 0.80902   -0.58779   0.00000
         // 0.58779    0.80902   0.00000
         // 0.00000    0.00000   1.00000
-        
+        q4.fromRotationMatrix(my);
+        q3.fromRotationMatrix(mx);
+        assertEqual(mx*my, (q3 * q4).toRotationMatrix(), "check quaternion multiplication", true);
```

Comment:
I would recommend to add the same test but with a more "complex" quaternion (no just the one obtained by the rotation on a cartesian axis).